cli_tools/google-osconfig-agent/config/config.go

```diff
@@ -27,8 +32,17 @@ import (
 const (
 	// InstanceMetadata is the instance metadata URL.
 	InstanceMetadata = "http://metadata.google.internal/computeMetadata/v1/instance"
-	// ReportURL is where OS configurations are written in guest attributes.
+	// ReportURL is the guest attributes endpoint.
```

Comment:
Can we rename this to `InventoryReportURL` to avoid confusing it with the patch report URL?

Reply:
We do use this for patch as well